data/hai missions.txt

```diff
@@ -1053,3 +1053,250 @@ mission "Hai Honeymoon"
+
+
+
+event "Nanachi: payment"
+
+
+
+mission "Nanachi: Payment"
+	landing
+	source
+		government "Hai"
+		not planet "Darkwaste"
+	to offer
+		has "event: Nanachi: payment"
+	on offer
+		payment 700000
+		event "Nanachi: Update" 720
+		conversation
+			`Shortly after you land, you get a message from Nanachi, the young hai pilot who you helped start out. "Dear captain <last>, I hope you're doing well! As I promised, ever since we parted ways I've been working towards getting enough money to pay you for being so kind to help me. I've sent you <payment>. I'm deeply grateful to you for helping make my dream come true."`
+				decline
+
+
+
+event "Nanachi: Update"
+
+
+
+mission "Nanachi Meeting"
+	landing
+	source
+		government "Hai"
+		not planet "Darkwaste"
+	to offer
+		has "event: Nanachi: Update"
+		random > 30
+	on offer
+		conversation
+			`After you leave your ship and start making your way through the docking area, you notice a Hai running towards you and waving. As they get closer, you realize that it's Nanachi, the Hai pilot who you helped out years ago. "I was just about to go back to my crew when I saw you coming out of your ship. I know a nice snack bar here, let's go there!" Before you can even say hello she's dragged you to the bar, and ordered some drinks for the two of you. She tells you how the last years have been, and of her experiences as a captain. She has long since sold her old ship, and now pilots a Geocoris, transporting large amounts of cargo across Hai space. You also tell her about how your life has been ever since you last met, of your adventures and how far you've come. You keep swapping stories for hours, until the bar is ready to close.`
+			`	You thank her for the drinks, and get ready to say your goodbyes. "Oh wait, before you go captain, let me give you something." She says, right as you're about to head to your ship, and grabs a small pendant from her pocket. "Back home in Giverstone, when I had some free time from the mining works, I took a hobby of sculpting some small stones into ship miniatures. I thought it'd be nice to make this for you in case I saw you again." She gives you the pendant, which has been sculpted into a Hai Shield Beetle, it is extremely well made. "It's just a good-luck charm I thought you'd like. Thanks again captain!" You two part ways, and head back to your ships.`
```

Comment:
`something." She` -> `something," She`

`your ship, and grabs` -> `your ship. She grabs`

`I thought it'd be nice to make this for you in case I saw you again."` But wasn't she a miner before she met you? How could she have sculpted this for you while she was still a miner?